app/components/Modal/SendModal.jsx

```diff
@@ -168,19 +168,47 @@ class SendModal extends React.Component {
             });
     }
 
+    fetchToAccount(toName) {
```

Comment:
Why did you make both fetchTo and fetchFrom, and then even with an accountName argument?

Reply:
changed